oak_containers/proto/launcher.proto

```diff
@@ -27,11 +27,20 @@ message GetImageResponse {
   bytes image_chunk = 1;
 }
 
+message GetContainerConfigResponse {
+  // Arbitrary config that the container can retrieve from the orchestrator.
+  // Included in the attestation measurements conducted by the orchestrator.
+  bytes config = 1;
+}
+
 service Launcher {
   // Provides stage1 with the Oak system image (which contains the Linux distribution and the
   // orchestrator binary).
   rpc GetOakSystemImage(google.protobuf.Empty) returns (stream GetImageResponse) {}
 
   // Provides orchestrator with the trusted container image.
   rpc GetContainerBundle(google.protobuf.Empty) returns (stream GetImageResponse) {}
+
+  // Provides orchestrator with the trusted container config.
+  rpc GetContainerConfig(google.protobuf.Empty) returns (GetContainerConfigResponse) {}
```

Comment:
some bikeshedding: My initial assumption about what "Container Config" implies was wrong (I thought this was the config.json for the container bundle). A different name (e.g. "Application Config) might make it clearer what the inteded use is.